content/en/security/application_security/risk_management/_index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Application Risk Management
+kind: documentation
+further_reading:
+- link: "/"
+  tag: "Documentation"
+  text: "tktk"
```

Comment:
Not sure if this is a placeholder for a link you intend to add later. Either way, making a note so it's visible to you.